### PR TITLE
Fixes for Glance/Cinder integration

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: chef/cookbooks/glance/attributes/default.rb [hunk 1]
```diff
@@ -63,11 +63,7 @@
 
 default[:glance][:sql_idle_timeout] = "3600"
 
-default[:glance][:glance_stores] = ["glance.store.filesystem.Store",
-                                    "glance.store.http.Store",
-                                    "glance.store.cinder.Store",
-                                    "glance.store.rbd.Store",
-                                    "glance.store.swift.Store"]
+default[:glance][:glance_stores] = ["file", "http", "cinder", "rbd", "s3"]
```
Comment (vuntz) [inline chef/cookbooks/glance/attributes/default.rb]: s/s3/swift/

**Comment by mjura** [comment]
Depends on https://github.com/crowbar/crowbar-openstack/pull/95

**Comment by dirkmueller** [comment]
@mjura why? its rather the other way around.. :)

**Comment by vuntz** [comment]
+1

**Comment by mjura** [comment]
+1

**Pull request merged**